system/jlib/jscm.hpp

```diff
@@ -211,4 +211,10 @@ interface jlib_thrown_decl IException : public IInterface
     virtual MessageAudience errorAudience() const = 0;
 };
 
+class jlib_decl EmbedCodeBlock
```

Comment:
Definitely hesitant about a dependency of generated code on a class defined in jlib.  I think it is probably ok, but it is the first instance.

Reply:
It is the first instance (other than people using SERVICE, LIBRARY(jlib)...) but it doesn't really create any new dependency (in that eclrtl is dependent on jlib).